docs/5g-on-helium/5g-on-helium.mdx

```diff
@@ -49,3 +49,45 @@ MOBILE tokens.
 [Economics Overview](https://github.com/helium/HIP/blob/main/0053-mobile-dao.md#economics-overview)
 in the community-approved HIP 53 and deposited to the Helium Foundation. For more information on the
 pre-mine and how it is used, see [Pre-Mine](/5g-on-helium/mobile-pre-mine).
+
+### Emissions Curve
+
+There will be a max supply of 250B (billion) MOBILE with issuance halvenings of every 2 years
+aligned with the HNT issuance halvenings. The first halving will occur on August 1, 2023, to align
+with HNT halving and will continue on a 2-year cycle afterward. This creates a "stub period" as
+defined by [HIP 53: Mobile subDAO](https://github.com/helium/HIP/blob/main/0053-mobile-dao.md) which
+will begin when normal emissions of MOBILE tokens begin (and MOBILE can be redeemed by HNT in the
+subDAO's treasury). The date for this is currently TBD and dependent on core developers implementing
+the HNT token treasury and redemption transactions on the Helium Blockchain. This is defined as
+Phase 2 of [HIP 51: Helium DAO](https://github.com/helium/HIP/blob/main/0051-helium-dao.md) for all
+new subDAOs.
+
+Before normal emissions of MOBILE begin,
+[HIP 51: Helium DAO](https://github.com/helium/HIP/blob/main/0051-helium-dao.md) defined an earlier
+launch of MOBILE tokens assigning a premine amount of MOBILE to the Helium Foundation. The Helium
+Foundation will work with the core developers to begin the Genesis period of MOBILE rewards. Genesis
+rewards will be paid out of the Foundation's pre-mined MOBILE rewards pool. Read more about the
+pre-mine distribution [here](/5g-on-helium/mobile-pre-mine).
+
+Currently, the Helium Foundation intends to issue 100M (million) tokens per day in the Genesis
+period, and it will be proportionally distributed to all active 5G Small Cell Radios as described
+below.
+
+## Manage Tokens with the Helium Wallet App
+
+- [App Store](https://apps.apple.com/us/app/helium-hnt-wallet/id1609525848)
+- [Google Play](https://play.google.com/store/apps/details?id=com.helium.wallet.app&gl=US)
+
+A 5G Hotspot with an attached and properly registered Small Cell Radio will start earning MOBILE
+automatically once the 5G rewards are activated on the Helium Blockchain.
+
+MOBILE Rewards are earned as one "lump sum" by a Wallet with a 5G Hotspot with at least one
+connected, SAS registered CBRS Small Cell Radio.
```

Comment:
An opportunity to reference MOBILE Proof of Coverage, by saying "You can learn more how rewards are calculated"